app/lib/frontend/handlers/pubapi.dart

```diff
@@ -336,6 +336,19 @@ class PubApi {
     await adminBackend.removeUser(userId);
     return jsonResponse({'status': 'OK'});
   }
+
+  @EndPoint.get('/api/admin/packages/<package>/flutter-favorite')
```

Comment:
Note for future API discussions: We have `/api/packages/<package>/options` for updating the `isDiscontinued` flag. I'd expect the admin API to also use `/options` but at the same time I understand the separation in this case. Once we have more APIs around, and before we mark them stable, we should do a bit of renaming to make them consistent.